VideoLocker/res/values/strings.xml

```diff
@@ -89,6 +89,8 @@
     <string name="label_close_drawer">Close Navigation</string>
     <!-- Button text to show user Settings screen -->
     <string name="action_settings">Settings</string>
+    <!-- Title of user's profile screen -->
+    <string name="label_my_profiles">My Profile</string>
```

Comment:
@catong FYI. Users cannot visually see this, it is only heard when using the screen reader.